src/graphnet/models/utils.py

```diff
@@ -15,3 +16,18 @@ def calculate_xyzt_homophily(x, edge_index, batch):
     hz = homophily(edge_index, x[:,2], batch).reshape(-1,1)
     ht = homophily(edge_index, x[:,3], batch).reshape(-1,1)
     return hx, hy, hz, ht
+
+
+def calculate_distance_matrix(xyz_coords):
+    '''
+    Input:
+        x.y,z co-ordinates of active DOMs
+
+    Calculates the matrix of distances squared from DOM i to DOM j in x,y,z co-ordinates
+
+    Returns:
+        matrix: tensor of size [N_doms,N_doms]
```

Comment:
```suggestion
    Calculate the matrix of pairwise distances between pulses in (x,y,z)-coordinates.
    
    Args:
        xyz_coords: (x,y,z)-coordinates of pulses, of shape [nb_doms, 3].

    Returns:
        Matrix of pairwise distances, of shape [nb_doms, nb_doms]
```

Reply:
To conform with standard (Google-style) docstring structure, see https://www.python.org/dev/peps/pep-0257/.